docs/modules/ROOT/pages/how-tos/testing/integration/snapshots/working-with-snapshots.adoc

```diff
@@ -0,0 +1,79 @@
+      source: <.>
+        git:
+          url: https://github.com/sample-org/sample-repo
+          revision: fa8b89274a61ef0f1c257b7a84c37aa2ec844109
+----
+<.> The name of the Snapshot resource.
+<.> The namespace where the Snapshot exists. It should correspond to the user's tenant namespace.
+<.> The Application that the Snapshot belongs to.
+<.> The name of the individual Component of the Application.
+<.> The full image pull specification for the container image. The images need to be referenced by digest.
+<.> The component source containing the git URL and revision that the component's container image was built from.
+
+== Working with manual Snapshots
+
+Aside from the Snapshots which are created and processed automatically by the {ProductName} services, it is also possible to create manual Snapshots which can then be tested and released if they conform to the users requirements.
+
+== Prerequisites
+- You have completed the steps listed in the xref:/getting-started/cli.adoc[Getting started in the CLI] page.
+- You have an existing managed workspace such as `ws-sample`, an application such as `application-sample` and a component such as `component-sample`.
+
+== Procedures
+To create a manual snapshot, complete the following steps:
+
+. Identify the application and component images which you want tested and released.
+. In your preferred IDE, create a snapshot definition in a `.yaml` file.
+.. For example contents, see the example Snapshot section above.
+. Save the .yaml file and add the snapshot.yaml by running the following command:
++
+[source,terminal]
+----
+$ kubectl create -f snapshot.yaml -n ws-sample-tenant
+----
+This command adds the Snapshot to your workspace and integration service will test and potentially trigger a release of the Snapshot.
+
+== Verification
+After integration service processes the created manual snapshot, you may verify snapshot and component by the following steps:
+
+. Check if Snapshot passed all tests by getting the `.Status.Conditions.AppStudioTestSucceeded` condition with the following command:
++
+[source,terminal]
+----
+$ kubectl get snapshot snapshot-sample -n ws-sample-tenant -o yaml | yq '.status.conditions.[] | select(.type =="AppStudioTestSucceeded")'
+lastTransitionTime: "2025-01-21T09:18:00Z"
+message: All Integration Pipeline tests passed
+reason: Passed
+status: "True"
+type: AppStudioTestSucceeded
+----
+. If you have xref:/advanced-how-tos/releasing/create-release-plan.adoc[automatic Releases defined], you can check if Snapshot was auto-released getting the `.Status.Conditions.AutoReleased` condition with the following command:
++
+[source,terminal]
+----
+$ kubectl get snapshot snapshot-sample -n ws-sample-tenant -o yaml | yq '.status.conditions.[] | select(.type =="AutoReleased")'
+lastTransitionTime: "2025-01-21T09:18:01Z"
+message: The Snapshot was auto-released
+reason: AutoReleased
+status: "True"
+type: AutoReleased
```

Comment:
Ah, you are verifying the release status of a snapshot. I can see how that might be relevant here. I feel like this might just need to be tied into the page earlier a little better?

Reply:
Ack, moved the information above.